### PR TITLE
integerate test mode

### PR DESCRIPTION
**Pull request by re-Tick**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,4547 @@
+{
```
Comment (rajatsharma) [inline package-lock.json:1]: Delete this file

File: package.json [hunk 2]
```diff
@@ -35,8 +36,11 @@
     }
   },
   "dependencies": {
+    "@fastify/express": "^1.1.0",
     "express": "^4.17.3",
     "got": "^11.5.2",
-    "require-in-the-middle": "^5.1.0"
+    "require-in-the-middle": "^5.1.0",
+    "fastify": "^3.29.0",
```
Comment (rajatsharma) [inline package.json:43]: Why are we adding **fastify** and **pino**?

**Review by rajatsharma — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (re-Tick) [reply package-lock.json]: Done.

File: package.json (hunk 2 above)
Comment (re-Tick) [reply package.json]: Oh forgot to delete it

**Comment by slayerjain** [comment]
@re-Tick whats up with this?

**Review by rajatsharma — APPROVED** [review]

**Review by slayerjain — APPROVED** [review]
LGTM

**Pull request merged**